### ISSUE TITLE
KeyError

### ISSUE THREAD

**Issue by DocKDE** [jensengroup/propka#48]
Hello,
I cloned the repo just now, followed the installation instructions and tried to run the program on a pdb file. This was the output:

`
Traceback (most recent call last):                                                                      
  File "/home/myuser/.local/bin/propka31", line 11, in <module>                                       
    load_entry_point('PROPKA==3.1.1', 'console_scripts', 'propka31')()                                  
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/run.py", line 18, in main                                                                                               
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/molecular_container.py", line 52, in __init__                                                                           
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/version.py", line 50, in setup_bonding_and_protonation
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/calculations.py", line 17, in setup_bonding_and_protonation
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/calculations.py", line 36, in setup_bonding
  File "/home/myuser/.local/lib/python3.6/site-packages/PROPKA-3.1.1-py3.6.egg/propka/bonds.py", line 44, in __init__
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1137, in resource_filename
    self, resource_name
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1646, in get_resource_filename
    return self._extract_resource(manager, zip_path)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1667, in _extract_resource
    timestamp, size = self._get_date_and_size(self.zipinfo[zip_path])
KeyError: 'propka/protein_bonds.json'
`

I'm afraid I don't know what exactly is wrong here and I would appreciate any assistance! 
I'm on Ubuntu 18.04 and python 3.6.7. If any more info is needed I'll be happy to provide it.

Best!

**Comment by sobolevnrm** [comment]
I am unable to reproduce this error.  Can you please share the command you entered prior to receiving this message?  Thank you.

**Comment by DocKDE** [comment]
I cd-ed to ~/.local/bin, copied a pdb file there (2bem.pdb) and entered:
`propka31 2bem.pdb` and
`./propka31 2bem.pdb`. I also tried to execute the propka31.py script in the scripts folder with:
`python3 propka31.py 2bem.pdb` which gave the same result.

**Comment by orbeckst** [comment]
Can you please show the exact commands that you used to install? E.g., I did 
```bash
# I use conda
conda create -n propka python=3.6 numpy pandas pytest
conda activate propka
git clone https://github.com/jensengroup/propka-3.1.git
pip install -v propka-3.1/
```
I will then test with
```
cd /tmp
curl https://files.rcsb.org/download//2bem.pdb.gz | gzip -dc > 2bem.pdb
propka31 2bem.pdb
```
... and I get the following error, similar to yours:
```

Traceback (most recent call last):
  File "~/Library/miniconda2/envs/propka/bin/propka31", line 8, in <module>
    sys.exit(main())
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/run.py", line 21, in main
    my_molecule = read_molecule_file(pdbfile, my_molecule)
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/input.py", line 63, in read_molecule_file
    mol_container.version.setup_bonding_and_protonation(mol_container)
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/version.py", line 88, in setup_bonding_and_protonation
    return self.molecular_preparation_method(molecular_container)
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/hydrogens.py", line 18, in setup_bonding_and_protonation
    my_bond_maker = setup_bonding(molecular_container)
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/hydrogens.py", line 37, in setup_bonding
    my_bond_maker = BondMaker()
  File "~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/bonds.py", line 44, in __init__
    with open(self.data_file_name, 'rt') as json_file:
FileNotFoundError: [Errno 2] No such file or directory: '~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/protein_bonds.json'
```

I think the `protein_bonds.json` file is not properly installed in `setup.py`.

**Comment by orbeckst** [comment]
```
ls ~/Library/miniconda2/envs/propka/lib/python3.6/site-packages/propka/
```
does not show the file
```
atom.py                    hybrid36.py           molecular_container.py
bonds.py                   hydrogens.py          output.py
calculations.py            __init__.py           parameters.py
conformation_container.py  input.py              propka.cfg
coupled_groups.py          iterative.py          protonate.py
determinant.py             lib.py                __pycache__
determinants.py            ligand_pka_values.py  run.py
energy.py                  ligand.py             vector_algebra.py
group.py                   __main__.py           version.py
```

**Comment by orbeckst** [comment]
I'll push a PR with a fix...

**Comment by orbeckst** [comment]
This was fixed in PR #61 .
